code-examples/protect-page-login/expressjs/index.js

```diff
@@ -0,0 +1,22 @@
+const app = require("express")()
+// highlight-next-line
+const sdk = require("@ory/client-fetch")
+
+// highlight-start
+const ory = new sdk.FrontendApi(
+  new sdk.Configuration({
+    basePath: process.env.ORY_ENDPOINT || "http://localhost:4000/.ory",
+  }),
+)
+ // highlight-end
+
+app.get("/", (req, res) =>
+  // highlight-start
+  ory
+    .toSession({ cookie: req.header("cookie") })
+    .then((data) => res.json(data))
```

Comment:
I like the minimalism of the example, but maybe something that is a tiny bit more human-readable would be nice here? :)

Reply:
Makes sense. Will update :)